src/register_op.cpp

```diff
@@ -34,6 +34,13 @@ std::unordered_map<std::string, operation>& op_map()
     return m;
 }
 void register_op(const operation& op) { op_map()[op.name()] = op; }
+
+void unregister_op(const std::string& op_name)
+{
+    assert(op_map().count(op_name));
```

Comment:
We shouldn't assert if its missing. It should be a no-op if it isn't register.

Reply:
`unregister_op` would only be called if the operator is registered first.  Therefore, IMO it is okay  to have an assert.